<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2560</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>timer</string>
  </property>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2560</width>
     <height>1600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Days1">
   <property name="geometry">
    <rect>
     <x>2032</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Days2">
   <property name="geometry">
    <rect>
     <x>2078</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Days3">
   <property name="geometry">
    <rect>
     <x>2124</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>40</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Hours1">
   <property name="geometry">
    <rect>
     <x>2200</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Hours2">
   <property name="geometry">
    <rect>
     <x>2245</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Minutes1">
   <property name="geometry">
    <rect>
     <x>2320</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Minutes2">
   <property name="geometry">
    <rect>
     <x>2365</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Seconds1">
   <property name="geometry">
    <rect>
     <x>2440</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Seconds2">
   <property name="geometry">
    <rect>
     <x>2485</x>
     <y>110</y>
     <width>45</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>2033</x>
     <y>10</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>距离2022考研还有</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>2178</x>
     <y>110</y>
     <width>45</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>2298</x>
     <y>110</y>
     <width>45</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>2418</x>
     <y>110</y>
     <width>45</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_background">
   <property name="geometry">
    <rect>
     <x>2021</x>
     <y>80</y>
     <width>520</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Resource/bg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_num_background</zorder>
  <zorder>label_background</zorder>
  <zorder>label_Days1</zorder>
  <zorder>label_Days2</zorder>
  <zorder>label_Days3</zorder>
  <zorder>label_Hours1</zorder>
  <zorder>label_Hours2</zorder>
  <zorder>label_Minutes1</zorder>
  <zorder>label_Minutes2</zorder>
  <zorder>label_Seconds1</zorder>
  <zorder>label_Seconds2</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_logo</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
